### DISCUSSION TITLE
please make access token last at least a year

### DISCUSSION THREAD

**Discussion by chrcoluk** [RetroAchievements/RAWeb#1813] [Ideas]
Also please consider allow copy and paste of password.

Having to relogin after two weeks, whilst having to also type it in means I will just change to a weak easy to type password.

**Comment by meleu** [comment]
+1 for the "allow copy and paste password". It should be a separate request, though.

**Comment by Kinglink** [comment]
Are you talking about the website?  I can copy and paste my password all the time.  I can use Tusk (Keypass solution), I can use Google Passwords to log in as well. 

If you can't copy and paste a password to login, it might be a problem on your browser.

**Comment by Alexdatadestroyer** [comment]
> Having to relogin after two weeks, whilst having to also type it in means I will just change to a weak easy to type password.

**If I am not mistaken**, on V1 Era, authentication data lasted for 7 days before needing to relogin. On V2, it was forever, but on V3 I noticed that it's now lasts for 1 month [?]. So, this is weird...

Sometimes, when a major version of RAWeb is released, you get instant logged off, but it's not so often though. (Last time, was when V3 came, if I recall correctly).

**Comment by chrcoluk** [comment]
To clarify the issue is me using retroarch on the password, it doesnt allow pasting.  My mistake there, as that probably a different repo.

Hopefully you guys can extend the token expiry. Also just noticed this is RAweb I apologise again :) Is it a different repo for the authentication token on emulators?
